install-rust.sh

```diff
@@ -19,20 +19,28 @@ set -x
 # rust-toolchain.toml. This file will pick it up automatically.
 RUST_VERSION=$(perl -ne 'if (/channel\s+=\s+"(\d+\.\d+)"/) { print $1 }' rust-toolchain.toml)
 
-# The host triple should basically never change. But just in case.
-HOST_TRIPLE=x86_64-unknown-linux-gnu
-
 # This is the SHA256 if the rustup-init binary (which is the same as rustup --
 # it renames itself) and should be retrieved from a trusted source (eg. the rust
 # website and/or by running sha256sum on a local copy of rustup you believe to
 # be legitimate). The canonical URL for the SHA256 checksum provided here is:
 #
 # https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-gnu/rustup-init.sha256
+# https://static.rust-lang.org/rustup/dist/aarach64-unknown-linux-gnu/rustup-init.sha256
```

Comment:
This looks good - basically we're saying that we only support "Tier 1" 64 bit Rust which is probably fine

Reply:
The only reason we need any of these hardcoded is because we want to have a list of sha's here. We can add as many as we want really, but these two should be the bare minimum since we only use this script in build environments which use Linux.